readme.md

```diff
@@ -38,22 +38,22 @@ For example in the *zsh* module both *antigen* (my personal preference) and *oh-
 Run the following commands:
 
 ```sh
-curl -L https://api.github.com/repos/mrjadaml/mac-maker/tarball --create-dirs -o ~/projects/dotfiles.tar.gz
+curl -L https://api.github.com/repos/MrJadaml/mac-maker/tarball --create-dirs -o ~/Projects/dotfiles.tar.gz
```

Comment:
does it not work with lowercase `mrjadaml`?

projects I like having lowercase on my computer. Just one less shift to press, and I'm super lazy.